CHANGELOG.md

```diff
@@ -0,0 +1,525 @@
+# Changelog
+
+## v5.0.0
```

Comment:
as long as it's generated, could we make this like:
```suggestion
## [v5.0.0](https://github.com/substack/tape/releases/tag/v5.0.0)
```
?

Reply:
Yes, we could